harness/tests/godot_kotlin_configuration.json

```diff
@@ -1 +1,14 @@
-{"vm_type":"jvm"}
+{
+  "debug_address": "*",
+  "debug_port": 5005,
+  "disable_closing_leaks_warning": false,
+  "disable_gc": false,
+  "force_gc": false,
+  "jmx_port": -1,
+  "jvm_args": "",
+  "max_string_size": -1,
```

Comment:
NIT; still -1? (I'm fine if you prefer it over 512 here)

Reply:
What do you mean still -1 ?
You disagreed with the default value being 0 as it people can want JNI strings only, so we decided to set it to -1 instead.

Reply:
fair